src/macro-core/macro-condition-window.cpp

```diff
@@ -40,7 +40,8 @@ static bool windowContainsText(const std::string &window,
 	return text == matchText;
 }
 
-bool MacroConditionWindow::WindowMatches(const std::string &window)
+bool MacroConditionWindow::WindowMatchesRequiremetns(
```

Comment:
Just a heads up you have a typo here `WindowMatchesRequiremetns`

Reply:
Thanks!